public/css/dashboard.css

```diff
@@ -34,91 +29,87 @@ main.dashboard .row {
 /* email card styling */
 .e-address {
   font-weight: 400;
-  margin-bottom: 0.5rem;
   font-size: 1.2rem;
 }
 
 .e-num-breaches {
   font-style: italic;
   font-weight: 300;
   color: #666;
+  margin-top: 0.5rem;
 }
 
 /* email card breaches */
-.email-card .e-breach-list {
+.e-breach-list {
   visibility: hidden;
   opacity: 0;
   max-height: 0;
   transition: all 0.3s ease;
 }
 
-.email-card.inactive .e-breach-list > * {
-  max-height: 0;
-  transition: all 0.3s ease;
-}
-
-.email-card.active .e-breach-list > * {
-  max-height: 10000px;
-  transition: all 0.3s ease;
-}
-
 .email-card.active .e-breach-list {
   visibility: visible;
   opacity: 1;
   max-height: 20000px;
   transition: all 0.3s ease;
 }
 
-.email-card.verified {
-  cursor: pointer;
+.email-card.active .e-breach-list > * {
+  max-height: 10000px;
```

Comment:
What is the "10000px" checking for? Looks almost arbitrary lookin'.

Reply:
This whole block is actually unnecessary now and defintiely almost arbitrary lookin' but the max-height on L53 is set to 20000px is so that we can smoothly transition the height of `.e-breach-list` from `max-height: 0` to `max-height: whatever-it-ends-up-being` since you can't just smoothly transition from `height: 0` to `height: auto`.